src/react-components/src/components/01-atoms/Button/Button.tsx

```diff
@@ -40,9 +33,9 @@ export interface ButtonProps {
   content?: string | JSX.Element;
   /** Additional attributes passed to the button */
   attributes?: {};
-  /** Used for BEM css convention. */
+  /** Optional modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
```

Comment:
I don't think you need these for every instance this comes up... It might be better to include this in the "intro" homepage for storybook and in the readme documentation.

Reply:
This suggestion came from me, as going forward I'd like to make sure we always have the info chart on components filled out:

![Screen Shot 2020-04-14 at 9 59 16 AM](https://user-images.githubusercontent.com/1720093/79233280-9d437900-7e36-11ea-8c6a-9c6b3a845a04.png)

I agree with you though, Edwin, that it would be nice if these were a link to longer documentation that describes BEM/how we're using it/etc. Those docs need to be written. Issue #109 tracks that.